package.json

```diff
@@ -1,5 +1,7 @@
 {
   "name": "content-data-admin",
   "private": true,
-  "dependencies": {}
+  "dependencies": {
+    "govuk-frontend": "^2.2.0"
```

Comment:
we don't need this bit, govuk-frontend is available through govuk_publishing_components. in fact I think you don't need this file at all